shopinvader_customer_validate/hooks.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from openupgradelib import openupgrade
+
+OLD_MODULE = "shopinvader_notification_default"
+NEW_MODULE = "shopinvader_customer_validate"
+RECORD_XMLIDS = [
+    # mail.template
+    "email_new_customer_welcome_not_validated_notification",
+    "email_customer_validated_notification",
+    "email_address_created_not_validated_notification",
+    "email_address_validated_notification",
+    "shopinvader_notification_new_customer_welcome_not_validated",
+    "shopinvader_notification_customer_validated",
+    "shopinvader_notification_address_created_not_validated",
+    "shopinvader_notification_address_validated",
+    # ir.ui.view
+    "address_created_not_validated",
+    "address_validated",
+    "new_customer_not_validated",
+    "customer_validated",
+]
+
+
+def pre_init_hook(cr):
+    update_record_xmlids(cr)
+
+
+def update_record_xmlids(cr):
+    old_module = "shopinvader_notification_default"
+    new_module = "shopinvader_customer_validate"
+    records_xmlids_spec = [
+        (f"{old_module}.{xmlid}", f"{new_module}.{xmlid}") for xmlid in RECORD_XMLIDS
+    ]
+    openupgrade.rename_xmlids(cr, records_xmlids_spec)
```

Comment:
is this fault tolerant in case the original xids are not there? (eg: you start from scratch w/ only the s_notification_default not including those templates)

Reply:
yup https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L948-L950